deploy/helm/prometheus-overrides.yaml

```diff
@@ -74,13 +74,13 @@ prometheus:
       - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.apiserver
         writeRelabelConfigs:
           - action: keep
-            regex: apiserver;(?:apiserver_request_count|apiserver_request_latencies.*|etcd_request_cache_get_latencies_summary.*|etcd_request_cache_add_latencies_summary.*|etcd_helper_cache_hit_count|etcd_helper_cache_miss_count)
+            regex: apiserver;(?:apiserver_request_(?:count|total)|apiserver_request_(?:latencies|duration_seconds).*|etcd_request_cache_get_(?:latencies_summary|duration_seconds).*|etcd_request_cache_add_(?:latencies_summary|duration_seconds).*|etcd_helper_cache_hit_(?:count|total)|etcd_helper_cache_miss_(?:count|total))
```

Comment:
should the etcd metrics here be `latency_seconds` not `duration_seconds`?

Reply:
same with `apiserver_request_duration_seconds`, should this be `apiserver_request_latency_seconds`? based on your description

Reply:
ah I should change the description. they changed `latency` to `duration` in those metrics between 1.14 and 1.16

Reply:
thanks for the confirmation!